editoast/src/views/speed_limit_tags.rs

```diff
@@ -0,0 +1,24 @@
+use actix_web::get;
+use actix_web::web::Json;
+
+use crate::error::Result;
+use crate::generated_data::speed_limit_tags_config::SpeedLimitTag;
+
+crate::routes! {
+    "/speed_limit_tags" => {
+        speed_limit_tags,
+    },
+}
+
+#[utoipa::path(
+    tag = "speed_limit_tags",
+    responses(
+        (status = 200, description = "List of configured speed-limit tags", body = Vec<String>, example = json!(["V200", "MA80"])),
+    ),
+)]
+#[get("")]
+async fn speed_limit_tags() -> Result<Json<Vec<String>>> {
+    let speed_limit_tags_config = SpeedLimitTag::load();
+    let speed_limit_tags = speed_limit_tags_config.keys().cloned().collect();
```

Comment:
I'm not sure that I understand, if only keys are used, why do we load in memory all the information. Will it be used later (in other endpoints)?

Reply:
Indeed, restricted exposed api to `Vec<String>` in https://github.com/OpenRailAssociation/osrd/pull/8106/commits/e379f7c4c2816f42cddc3c79360b2d20e330beab
Kept the whole structure to "make sure" the file and deserializer are synced when reading it.